### PR TITLE
Bulgarian Translation 14

### PR DESCRIPTION
**Pull request by Effektus**
400 Rows Left

### CODE REVIEW

File: files/lang/bg.po [hunk 1]
```diff
@@ -5221,69 +5221,91 @@ msgid ""
 "The lighthouse is now under your control, and all of your ships will now "
 "move further each day."
 msgstr ""
+"Фарът вече е под твой контрол и всичките ти кораби ще се придвижват всеки "
```
Comment (zenseii) [inline files/lang/bg.po:5224]: Using deepl it seems that this says that your ships will move every day and not that they will move further every day? Please ignore this if it's wrong.

File: files/lang/bg.po [hunk 2]
```diff
@@ -5293,140 +5315,196 @@ msgid ""
...
 "An unusual alliance of Ogres, Orcs, and Dwarves offer to train (upgrade) any "
 "such troops brought to them. Unfortunately, you have none with you."
 msgstr ""
+"Необичаен съюз на Огри, Орки и Джуджета предлага да обучава (надгражда) "
+"всички войски, които са им доведени. За съжаление нямаш нито една със себе "
+"си."
 
 msgid "All of your %{monsters} have been upgraded into %{monsters2}."
-msgstr ""
+msgstr "Всичките ти %{monsters} са надградени до %{monsters 2}."
```
Comment (zenseii) [inline files/lang/bg.po:5507]: Please remove the white space from here.
```suggestion
msgstr "Всичките ти %{monsters} са надградени до %{monsters2}."
```

File: files/lang/bg.po [hunk 3]
```diff
@@ -5435,133 +5513,181 @@ msgid ""
...
-msgstr ""
+msgstr "(Просто го зарови около корените ми.)"
 
 msgid "Tears brim in the eyes of the tree."
-msgstr ""
+msgstr "Сълзи бликват от очите на дървото."
 
 msgid "\"I need %{count} %{res}.\""
-msgstr ""
+msgstr "„Имам нужда от %{count} %{res}.“"
```
Comment (zenseii) [inline files/lang/bg.po:5592]: Please use different quotation marks.

**Review by zenseii — CHANGES_REQUESTED** [review]
Hi, @Effektus.

I left some few comments. Please have a look should you have the possibility.

**Review by zenseii — APPROVED** [review]

File: files/lang/bg.po [hunk 4]
```diff
@@ -5293,140 +5315,196 @@ msgid ""
 "whisper: \"Why have you come? Are you here to call upon the forces of the "
 "air?\""
 msgstr ""
+"Бели каменни стълбове поддържат покрив, който се издига до небето. Когато "
+"влизаш в структурата, мъртвият въздух отвън отстъпва място на въртящ се "
+"порив, който почти те избутва обратно. Въздушното течение се материализира в "
+"едва видима форма. Съществото пита, което може да се опише само като силен "
+"шепот: „Защо си дошъл? Тук си, за да призовеш силите на въздуха ли?“"
```
Comment (ihhub) [inline files/lang/bg.po:5322]: Hi @zenseii , do we have these quotation marks for Bulgarian? If not then we have to use `\"` symbol for these cases?

Comment (ihhub) [reply files/lang/bg.po:5322]: @Effektus and @zenseii , I believe we should use `\"` as it is done everywhere else in the code.

File: files/lang/bg.po [hunk 5]
```diff
@@ -5293,140 +5315,196 @@ msgid ""
...
+msgstr "Лице се оформя във водата за момент и след това изчезва."
 
 msgid ""
 "Crystalline structures cast shadows over a small reflective pool of water. "
 "You peer into the pool, and a face that is not your own peers back. It asks: "
 "\"Would you like to call upon the powers of water?\""
 msgstr ""
+"Кристални структури хвърлят сенки върху малък отразяващ воден басейн. "
+"Надникваш в басейна и лице, което не е твоето, се оглежда там. То пита: "
+"„Искаш ли да призовеш силите на водата?“"
```
Comment (ihhub) [inline files/lang/bg.po:5346]: The same comment here.

File: files/lang/bg.po (hunk 4 above)
Comment (zenseii) [reply files/lang/bg.po:5322]: Yep, I missed these.

**Comment by ihhub** [comment]
Hi @Effektus , I left 2 comments here. Could you please take a look?

**Comment by ihhub** [comment]
@Effektus , a huge thanks for polishing this translation!

**Pull request merged**